src/ui/ResponsiveComponents/ResponsiveComponentsManager.ts

```diff
@@ -32,7 +33,21 @@ interface IComponentInitialization {
   arguments: [IResponsiveComponentConstructor, Dom, string, Component, IResponsiveComponentOptions];
 }
 
+/**
```

Comment:
I would really prefer not to expose this globally/documented like that. 

I know it can be annoying for integrations that we dont export everything globally always all the time, but there's a good reason for this...

By commenting this and exposing it publicly, this means we're bound "by contract" to never break any backward compatibility on any method signature, because this is no longer an internal module, it can be used and referenced directly by clients.

I would vote to instead expose a single method on the search interface `resizeAllComponentsManager` that calls the proper module/function underneath.

Comment only the search interface method publicly. Uncomment/un-export this module.

By just doing this simple change, we've at least reduced the surface area quite a bit.

Reply:
I understand your point, having more stuff public is never a trifling matter.
I like your suggestion, I'll do just that!